hack/common.sh

```diff
@@ -70,7 +70,7 @@ readonly -f os::build::platform_arch
 function os::build::setup_env() {
   os::util::ensure::system_binary_exists 'go'
 
-  if os::util::find::system_binary 'sha256sum' >/dev/null 2>&1; then
+  if [[ -z "$(which sha256sum)" ]]; then
```

Comment:
I guess this is technically not the same as `os::util::find::system_binary` .... but that was the _intent_ behind this check, right? If we don't have the program, declare a no-op. Re-declaring when we've already declared is fine.